src/components/CompareResults/beta/RevisionHeader.tsx

```diff
@@ -38,10 +42,38 @@ const styles = {
   }),
 };
 
-function createTitle(header: RevisionsHeader) {
-  return header.test === '' || header.suite === header.test
-    ? header.suite
-    : `${header.suite} ${header.test}`;
+function createTitle(
+  header: RevisionsHeader,
+  docsURL: string,
+  isLinkSupported: boolean,
+) {
+  const isTestUnavailable = header.test === '' || header.suite === header.test;
+  if (isLinkSupported) {
+    return isTestUnavailable ? (
+      <Link
+        aria-label='link to suite documentation'
+        underline='hover'
+        target='_blank'
+        href={docsURL}
+      >
+        {header.suite}
+      </Link>
+    ) : (
+      <>
+        <Link
+          aria-label='link to suite documentation'
+          underline='hover'
+          target='_blank'
+          href={docsURL}
+        >
+          {header.suite}
+        </Link>
+        &nbsp;{header.test}
```

Comment:
You could move the condition for `isTestUnavailable` here. 
There will always be a suite available, only the test name can be missing.